app/controllers/api/v1/cars_controller.rb

```diff
@@ -21,6 +21,27 @@ def create
     end
   end
 
+  # PUT /cars/:id
+  def update
+    if @car
+      @car.update(car_params)
```

Comment:
Please, can you confirm what happens if the update method fails? According to the code, the client will receive a success message and only receives an error message if the car doesn't exist.

Reply:
Yes!!! You're right, I'm improving it

Reply:
I fixed. Now when you tried to update a car that doesn't exits the api answer with a message:

![image](https://user-images.githubusercontent.com/10566417/236586474-9eb97c14-8dcc-4ebc-adc1-d69f6462831c.png)

And when you tried to update an attribute that doesn't exists the api answer with another message:

![image](https://user-images.githubusercontent.com/10566417/236586548-bc495800-6268-4fbd-af69-6b4555b0989a.png)

And when you tried to update an atribute with an invalid value the api answer with another message:

![image](https://user-images.githubusercontent.com/10566417/236586780-101a2136-01af-4ec0-ac79-5ead4512c685.png)